### PR TITLE
Remove hash

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Remove hash from navbar items
2. Update the styling of spotlight card
3. Maintain the same height for project cards
4. Increase the size of close icon of alert box
5. Add pace.js to show progress bar
6. Update the font size of project section headlines
7. Update the icon size of tabs since the tap area was small

Dependency: https://github.com/hasgeek/baseframe/pull/298

### CODE REVIEW

File: funnel/assets/js/app.js [hunk 1]
```diff
@@ -83,3 +83,16 @@ $(() => {
     $.cookie('timezone', jstz.determine().name(), { path: '/' });
   }
 });
+
+if (
+  navigator.userAgent.match(/(iPhone|Android)/) &&
```
Comment (jace) [inline funnel/assets/js/app.js:88]: This may need `iPad` also.

File: funnel/assets/js/util.js [hunk 2]
```diff
@@ -38,8 +38,7 @@ export const Utils = {
     );
   },
   smoothScroll() {
-    $('a.js-smooth-scroll').on('click', function clickHandler(event) {
-      event.preventDefault();
+    $('a.js-smooth-scroll').on('click', function clickHandler() {
       Utils.animateScrollTo($(this.hash).offset().top);
```
Comment (jace) [inline funnel/assets/js/util.js:42]: What does this do?

Comment (vidya-ram) [reply funnel/assets/js/util.js:42]: Add the hash to the url for tabs. For ex: `#about`, `#talks`, `#venue` on the project page

File: funnel/assets/js/app.js (hunk 1 above)
Comment (vidya-ram) [reply funnel/assets/js/app.js:88]: iPad browser doesn't have a scrollbar so we can keep the pace scrollbar

**Pull request merged**